editor/gui/editor_validation_panel.cpp

```diff
@@ -64,9 +64,11 @@ void EditorValidationPanel::add_line(int p_id, const String &p_valid_message) {
 	ERR_FAIL_COND(valid_messages.has(p_id));
 
 	Label *label = memnew(Label);
-	message_container->add_child(label);
 	label->set_custom_minimum_size(Size2(200 * EDSCALE, 0));
 	label->set_autowrap_mode(TextServer::AUTOWRAP_WORD_SMART);
+	// Must add the Label node as a child only after setting the min size and
+	// autowrap, otherwise it explodes and becomes thousands of pixels tall.
```

Comment:
I think this comment is unnecessary.
I used to add node and then configure it, but realized it was a mistake. We use either pattern pretty interchangeably without comments. It's not like someone is going to change it again.